docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md

```diff
@@ -29,9 +31,16 @@ Following are the [Fields](/docs/manage/fields/) which will be created as part o
 - **`db.system`**. Has fixed value of **mongodb**.
 - **`deployment.environment`**. User configured. This is the deployment environment where the Mongodb cluster resides. For example: dev, prod or qa.
 - **`sumo.datasource`**. has a fixed value of **mongodb**.
+* **`db.node.name`**. Has the value of host name of the machine which is being monitored.
 
 ## Prerequisites
 
+### For metrics collection
+
+The MongoDB receiver fetches stats from a MongoDB instance using the [golang mongo driver](https://github.com/mongodb/mongo-go-driver). Stats are collected via MongoDB's `dbStats` and `serverStatus` commands. The MongoDB receiver extracts values from the result and converts them to OpenTelemetry metrics.
+
+### For logs collection
```

Comment:
Will need to add instruction for profiling level setting in mongo for sending logs.

Reply:
it is already there on line number 46

Reply:
That is log level. We need to add instruction for setting profiling level